pkg/endpoint/endpoint.go

```diff
@@ -1294,6 +1321,38 @@ func (e *Endpoint) GetPod() *slim_corev1.Pod {
 	return e.pod.Load()
 }
 
+// CEPOwnerInterface contains the interface of an endpoint owner.
```

Comment:
How about adding a note: by default it's the pod information associated with an endpoint? There is no other reference to endpoint owner in the code.